### PR TITLE
Fixed endpoints

### PR DESCRIPTION
**Pull request by vivianneyee**
- endpoints return object attributes (where appropriate)
- added endpoints to add posts (event, assignment, alert) to class

### CODE REVIEW

**Review by viviennecruz — APPROVED** [review]

**Comment by vivianneyee** [comment]
I'm going to wait until the login/signup ui PR is merged so I can test the endpoints with the fix from that PR

**Pull request merged**
